hack/update-generated-completions.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source "${STI_ROOT}/hack/common.sh"
+
+sti::build::build_binaries "$@"
+
+echo "+++ Updating Bash completion (${STI_ROOT}/contrib/bash/s2i)"
```

Comment:
i don't really have any strong preference :-) i can remove it, I thought it would be nice to show where the completion was generated for debugging.